examples/jsm/controls/OrbitControls.js

```diff
@@ -1206,13 +1220,13 @@ class OrbitControls extends EventDispatcher {
 
 		}
 
-		//
+		if (_domElement) {
+
+			console.warn( 'THREE.OrbitControls: The domElement constructor param has been deprecated. Use .connect() instead.' );
```

Comment:
I would remove this warning so both approaches to setup `OrbitControls` are supported (meaning with or without using `connect()` on app level).

Reply:
Sure, I can do that! Generally I prefer APIs just having 1 way of doing a thing, mostly because of the reduced complexity for docs/test etc. But in this case it might cause too much churn for existing users to upgrade